data_model/src/permissions.rs

```diff
@@ -75,9 +79,32 @@ impl Registered for PermissionTokenDefinition {
 }
 
 impl PermissionTokenDefinition {
-    /// Construct new `PermissionTokenDefinition`
-    pub fn new(id: Id) -> Self {
-        Self { id }
+    /// Construct new [`PermissionTokenDefinition`]
+    #[inline]
+    pub fn new(id: <PermissionTokenDefinition as Identifiable>::Id) -> Self {
+        Self {
+            id,
+            params: btree_map::BTreeMap::default(),
+        }
+    }
+
+    /// Add parameters to the `PermissionToken` replacing any previously defined
```

Comment:
params are added to the definition (`PermissionTokenDefinition`), right?

```suggestion
    /// Add parameters to the `PermissionTokenDefinition` replacing any parameters previously defined.
```